source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst

```diff
@@ -1,126 +1,176 @@
 .. _increase-virtual-disk-size:
 
-Increase Clear Linux image virtual disk size
-############################################
+Increase virtual disk size of a Clear Linux image 
+#################################################
 
-Prebuilt |CLOSIA| images come in different sizes, ranging from 300 MB to 20
+|CLOSIA| prebuilt images come in different sizes, ranging from 300 MB to 20
 GB. This guide describes how to increase the size of your prebuilt |CL| image
 if you need more capacity.
 
-Determine the prebuilt image size
-*********************************
+.. contents:: This guide will cover:
 
-There are two methods to find the virtual disk size of your prebuilt |CL|
+
+Determine the partition order and sizes of the prebuilt image 
+***************************************************
+
+There are two methods to find the order and sizes of partitions virtual disk of your prebuilt |CL|
 image.
 
-The first method is to check the config.JSON file of the image, located in the
+In both examples, the prebuilt Hyper-V image 
+has a disk size of 8.5 GB with /dev/sdd3 being the partition
+for the root filesystem (/)
+
+
+Checking :command:`lsblk` on the VM
+-----------------------------------
+
+The first method is to boot up your :abbr:`VM (Virtual Machine)` and
+execute the :command:`lsblk` command as shown below:
+
+    .. code-block:: bash
+
+        sudo lsblk
+
+An example output of the :command:`lsblk` command: 
+
+    .. code-block:: console
+
+        NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+        sda      8:0   0    8.5G  0 disk
+        ├─sdd1   8:1   0    512M  0 part
```

Comment:
Shouldn't this line and the two below reference partitions from `sda` instead of `sdd`, since they are listed under `sda`? For example, I would expect this line to reference `sda1` instead.

Reply:
@puneetse I have several comments and corrections to syntax. Can you check in with me in person?

Reply:
@phmccarty Hm, good catch. I'm not sure where that original output was captured from or why it was modified but you're right. I verified on another system to make sure sdd isn't referenced by default. 

@mvincerx done.